### PR TITLE
Empty default ocp image (taking assisted-service default)

### PR DESCRIPTION
**Pull request by YuviGold**

### CODE REVIEW

**Review by gamli75 — APPROVED** [review]

File: Jenkinsfile [hunk 1]
```diff
@@ -10,7 +10,7 @@ pipeline {
         string(name: 'SERVICE_REPO', defaultValue: 'https://github.com/openshift/assisted-service', description: 'assisted service repository')
         string(name: 'IGNITION_GENERATE_IMAGE', defaultValue: '', description: 'assisted-ignition-generator image')
         string(name: 'INSTALLER_IMAGE', defaultValue: '', description: 'installer image to use')
-        string(name: 'OPENSHIFT_INSTALL_RELEASE_IMAGE', defaultValue: 'quay.io/ocpmetal/ocp-release:4.6.0-0.nightly-2020-08-31-220837', description: 'OCP Release Image from ocpmetal repository in Quay.io')
+        string(name: 'OPENSHIFT_INSTALL_RELEASE_IMAGE', defaultValue: '', description: 'OCP Release Image from ocpmetal repository in Quay.io')
```
Comment (tsorya) [inline Jenkinsfile:13]: didn't knew we had this one. Not sure why we had default value before

Comment (YuviGold) [reply Jenkinsfile:13]: yap https://github.com/openshift/assisted-test-infra/pull/156

File: discovery-infra/update_assisted_service_cm.py [hunk 2]
```diff
@@ -36,6 +36,7 @@
     ("HARDWARE_INFO_IMAGE", ""),
     ("INVENTORY_IMAGE", ""),
     ("OCM_BASE_URL", ""),
+    ("OPENSHIFT_INSTALL_RELEASE_IMAGE", "")
```
Comment (jparrill) [inline discovery-infra/update_assisted_service_cm.py:39]: @YuviGold If you set this one as it is, the ConfigMap template (source file) will empty the REPLACE_XXXXX string, then the replacement will not happen on the Assisted-Service repository

File: Jenkinsfile (hunk 1 above)
Comment (ronniel1) [reply Jenkinsfile:13]: Did you verify that the Makefile of assisted-service overrides this?

**Pull request merged**